### PR TITLE
Feat: Cart Management

### PR DESCRIPTION
**Pull request by yogesh-**
Completed cart management page .

image

Preview link - https://deploy-preview-3--sneaker-don.netlify.app/

### CODE REVIEW

File: css/cart.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+@import url(/css/styles.css);
+
+.cart-management {
+  display: grid;
+  grid-template-areas:
+    "nav nav nav nav"
+    "heading heading heading heading"
+    "empty card price empty2"
```
Comment (akash-1318) [inline css/cart.css:8]: These names can be more elegant

Comment (lannisterrr) [inline css/cart.css:8]: You can use "." instead of declaring the empty div. and writing empty here.

**Review by lannisterrr — COMMENTED** [review]
other than points mentioned above , the code seems fine.

**Review by anjali1102 — APPROVED** [review]
LGTM !

**Review by hiteshverma27 — APPROVED** [review]
looks good

File: css/cart.css (hunk 1 above)
Comment (yogesh-) [reply css/cart.css:8]: there was some error and the block was not appearing.
figuring our.

Comment (yogesh-) [reply css/cart.css:8]: okay noted.

**Pull request merged**